tests/appsec/api_security/test_schemas.py

```diff
@@ -48,7 +48,9 @@ def test_request_method(self):
         assert self.request.status_code == 200
         assert schema
         assert isinstance(schema, list)
-        assert contains(schema, [{"accept-encoding": [8], "host": [8], "user-agent": [8]}])
+        for parameter_name in ("accept-encoding", "host", "user-agent"):
```

Comment:
We should consider to move this kind of assertion inside schema on day.

Reply:
will resolve for now, but sounds good, for another PR move to https://github.com/DataDog/system-tests/tree/main/utils/interfaces/schemas et describe schemas in json schemas files.